packages/suins/sources/suins.move

```diff
@@ -143,18 +158,34 @@ public fun app_add_balance<App: drop>(
     self: &mut SuiNS,
     balance: Balance<SUI>,
 ) {
-    assert_app_is_authorized<App>(self);
+    self.assert_app_is_authorized<App>();
     self.balance.join(balance);
 }
 
+public fun app_add_balance_v2<App: drop, T>(
+    _: App,
+    self: &mut SuiNS,
```

Comment:
Let's correct our mistakes and make `&mut SuiNS` the first argument here so receiver syntax works!